### PR TITLE
[WebApp] Remove login functionality & add redirect to main login

### PR DESCRIPTION
**Pull request by rners01**
**Description:**  Remove login functionality & add redirect to main login
**Notion Task:** https://www.notion.so/saladtech/App-Remove-the-login-from-web-app-5b8064fdff384be5a6c6ad52d1db2af2
**Notion Project:** https://www.notion.so/saladtech/Frontend-Remove-web-app-login-pages-e7159a0e89fb4da9a05e17deda7049f5

### CODE REVIEW

File: packages/web-app/src/modules/auth/AuthStore.tsx [hunk 1]
```diff
@@ -61,23 +21,9 @@ export class AuthStore {
...
-        this.loginReject = reject
-
-        // Save current route so we can go back to it later once it is resolved
-        this.startingRoute = this.router.location.pathname === '/login' ? '/store' : this.router.location.pathname
-
-        this.router.replace('/login')
-      })
-
-      await this.loginPromise
+      window.location.replace(`${config.loginUrl}?redirect_webapp=${routeWithoutTrailingSlash}`)
```
Comment (Maks19) [inline packages/web-app/src/modules/auth/AuthStore.tsx:26]: Why did you use window.location.replace? 

```suggestion
      this.router.replace(`/login?redirect_webapp=${routeWithoutTrailingSlash}`)
```

Comment (rners01) [reply packages/web-app/src/modules/auth/AuthStore.tsx:26]: it will not work locally

Comment (Maks19) [reply packages/web-app/src/modules/auth/AuthStore.tsx:26]: Even if I run Login app locally?

Comment (rners01) [reply packages/web-app/src/modules/auth/AuthStore.tsx:26]: you can't run 2 apps on the same port

Comment (rners01) [reply packages/web-app/src/modules/auth/AuthStore.tsx:26]: but also for the development it doesn't make sense to always start login app along with web

Comment (rners01) [reply packages/web-app/src/modules/auth/AuthStore.tsx]: @Maks19 changed

**Pull request merged**